initFlutterSample.sh

```diff
@@ -11,7 +11,7 @@ flutter create -a java --org com.microblink $appName
 # enter into demo project folder
 pushd $appName
 
-IS_LOCAL_BUILD=false || exit 1
```

Comment:
@mijo-gracanin @krizaa jel bi ovaj flag trebao vratit nazad na false?

Reply:
Da, to je samo kada lokalno testiraš lib. Želimo da se sample izbuilda s libom koji se skine s https://pub.dev/packages/blinkid_flutter.